Semantic release build is failing with "OnClickXmlDetector called context.getMainProject()"

#### Summary

This commit:
https://github.com/googlemaps/android-maps-utils/commit/e6276a9eba798a9f56f34d1d0809cba36cfecf97

...from this PR:
https://github.com/googlemaps/android-maps-utils/pull/972

...succeeds in the normal Android build on CI:
https://github.com/googlemaps/android-maps-utils/runs/3603679874

...but fails in the semantic release build:
https://github.com/googlemaps/android-maps-utils/actions/runs/1235277089

`com.google.android.gms:play-services-base` was bumped as part of the PR, not sure if that's the source of the issue. Curiously dependabot never bumped this in `main`, but it was bumped in `master` (presumably during the switch).

#### Expected behavior

Release build should succeed

#### Observed behavior

Release build fails

#### Environment details

GitHub CI (see above)

#### Steps to reproduce

Look at failed CI build:
https://github.com/googlemaps/android-maps-utils/runs/3603679877

#### Stack trace

The log is too big to view in the GitHub UI, but from the raw log it looks like a lint issue?

```
2021-09-14T21:20:35.5018990Z     'w: Some runtime JAR files in the classpath have an incompatible version. Consider removing them from the classpath\n' +
2021-09-14T21:20:35.5020220Z     'Lint found errors in the project; aborting build.\n' +
2021-09-14T21:20:35.5020935Z     '\n' +
2021-09-14T21:20:35.5021941Z     'Fix the issues identified by lint, or add the following to your build script to proceed with errors:\n' +
2021-09-14T21:20:35.5022850Z     '...\n' +
2021-09-14T21:20:35.5023417Z     'android {\n' +
2021-09-14T21:20:35.5024062Z     '    lintOptions {\n' +
2021-09-14T21:20:35.5024791Z     '        abortOnError false\n' +
2021-09-14T21:20:35.5025419Z     '    }\n' +
2021-09-14T21:20:35.5025941Z     '}\n' +
2021-09-14T21:20:35.5026464Z     '...\n' +
2021-09-14T21:20:35.5026983Z     '\n' +
2021-09-14T21:20:35.5027718Z     'FAILURE: Build failed with an exception.\n' +
2021-09-14T21:20:35.5028405Z     '\n' +
2021-09-14T21:20:35.5029002Z     '* What went wrong:\n' +
2021-09-14T21:20:35.5029837Z     "Execution failed for task ':demo:lintGmsDebug'.\n" +
2021-09-14T21:20:35.5031910Z     '> A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintTask$AndroidLintLauncherWorkAction\n' +
2021-09-14T21:20:35.5034045Z     '   > There was a failure while executing work items\n' +
2021-09-14T21:20:35.5035985Z     '      > A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintWorkAction\n' +
2021-09-14T21:20:35.5037875Z     '         > Lint found errors in the project; aborting build.\n' +
2021-09-14T21:20:35.5038594Z     '\n' +
2021-09-14T21:20:35.5039365Z     '* Try:\n' +
2021-09-14T21:20:35.5040384Z     'Run with --debug option to get more log output. Run with --scan to get full insights.\n' +
2021-09-14T21:20:35.5041370Z     '\n' +
2021-09-14T21:20:35.5042082Z     '* Exception is:\n' +
2021-09-14T21:20:35.5043573Z     "org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':demo:lintGmsDebug'.\n" +
2021-09-14T21:20:35.5055201Z     '\tat org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.lambda$executeIfValid$1(ExecuteActionsTaskExecuter.java:187)\n' +
2021-09-14T21:20:35.5058119Z     '\tat org.gradle.internal.Try$Failure.ifSuccessfulOrElse(Try.java:268)\n' +
2021-09-14T21:20:35.5061413Z     '\tat org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.executeIfValid(ExecuteActionsTaskExecuter.java:185)\n' +
2021-09-14T21:20:35.5065916Z     '\tat org.gradle.api.internal.tasks.execution.ExecuteActionsTaskExecuter.execute(ExecuteActionsTaskExecuter.java:173)\n' +
2021-09-14T21:20:35.5070123Z     '\tat org.gradle.api.internal.tasks.execution.CleanupStaleOutputsExecuter.execute(CleanupStaleOutputsExecuter.java:109)\n' +
2021-09-14T21:20:35.5074921Z     '\tat org.gradle.api.internal.tasks.execution.FinalizePropertiesTaskExecuter.execute(FinalizePropertiesTaskExecuter.java:46)\n' +
2021-09-14T21:20:35.5079809Z     '\tat org.gradle.api.internal.tasks.execution.ResolveTaskExecutionModeExecuter.execute(ResolveTaskExecutionModeExecuter.java:51)\n' +
2021-09-14T21:20:35.5084475Z     '\tat org.gradle.api.internal.tasks.execution.SkipTaskWithNoActionsExecuter.execute(SkipTaskWithNoActionsExecuter.java:57)\n' +
2021-09-14T21:20:35.5088781Z     '\tat org.gradle.api.internal.tasks.execution.SkipOnlyIfTaskExecuter.execute(SkipOnlyIfTaskExecuter.java:56)\n' +
2021-09-14T21:20:35.5092790Z     '\tat org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:36)\n' +
2021-09-14T21:20:35.5096638Z     '\tat org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.executeTask(EventFiringTaskExecuter.java:77)\n' +
2021-09-14T21:20:35.5100076Z     '\tat org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.call(EventFiringTaskExecuter.java:55)\n' +
2021-09-14T21:20:35.5103452Z     '\tat org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.call(EventFiringTaskExecuter.java:52)\n' +
2021-09-14T21:20:35.5107271Z     '\tat org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:200)\n' +
2021-09-14T21:20:35.5111437Z     '\tat org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:195)\n' +
2021-09-14T21:20:35.5115313Z     '\tat org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:75)\n' +
2021-09-14T21:20:35.5118677Z     '\tat org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:68)\n' +
2021-09-14T21:20:35.5122293Z     '\tat org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:153)\n' +
2021-09-14T21:20:35.5126066Z     '\tat org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:68)\n' +
2021-09-14T21:20:35.5129743Z     '\tat org.gradle.internal.operations.DefaultBuildOperationRunner.call(DefaultBuildOperationRunner.java:62)\n' +
2021-09-14T21:20:35.5133548Z     '\tat org.gradle.internal.operations.DefaultBuildOperationExecutor.lambda$call$2(DefaultBuildOperationExecutor.java:76)\n' +
2021-09-14T21:20:35.5138290Z     '\tat org.gradle.internal.operations.UnmanagedBuildOperationWrapper.callWithUnmanagedSupport(UnmanagedBuildOperationWrapper.java:54)\n' +
2021-09-14T21:20:35.5142928Z     '\tat org.gradle.internal.operations.DefaultBuildOperationExecutor.call(DefaultBuildOperationExecutor.java:76)\n' +
2021-09-14T21:20:35.5146774Z     '\tat org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter.execute(EventFiringTaskExecuter.java:52)\n' +
2021-09-14T21:20:35.5150181Z     '\tat org.gradle.execution.plan.LocalTaskNodeExecutor.execute(LocalTaskNodeExecutor.java:74)\n' +
2021-09-14T21:20:35.5153661Z     '\tat org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$InvokeNodeExecutorsAction.execute(DefaultTaskExecutionGraph.java:408)\n' +
2021-09-14T21:20:35.5157512Z     '\tat org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$InvokeNodeExecutorsAction.execute(DefaultTaskExecutionGraph.java:395)\n' +
2021-09-14T21:20:35.5161575Z     '\tat org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareExecutionAction.execute(DefaultTaskExecutionGraph.java:388)\n' +
2021-09-14T21:20:35.5165797Z     '\tat org.gradle.execution.taskgraph.DefaultTaskExecutionGraph$BuildOperationAwareExecutionAction.execute(DefaultTaskExecutionGraph.java:374)\n' +
2021-09-14T21:20:35.5169279Z     '\tat org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.lambda$run$0(DefaultPlanExecutor.java:127)\n' +
2021-09-14T21:20:35.5171955Z     '\tat org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.execute(DefaultPlanExecutor.java:191)\n' +
2021-09-14T21:20:35.5174793Z     '\tat org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.executeNextNode(DefaultPlanExecutor.java:182)\n' +
2021-09-14T21:20:35.5177566Z     '\tat org.gradle.execution.plan.DefaultPlanExecutor$ExecutorWorker.run(DefaultPlanExecutor.java:124)\n' +
2021-09-14T21:20:35.5180177Z     '\tat org.gradle.execution.plan.DefaultPlanExecutor.process(DefaultPlanExecutor.java:72)\n' +
2021-09-14T21:20:35.5183775Z     '\tat org.gradle.execution.taskgraph.DefaultTaskExecutionGraph.executeWithServices(DefaultTaskExecutionGraph.java:193)\n' +
2021-09-14T21:20:35.5187902Z     '\tat org.gradle.execution.taskgraph.DefaultTaskExecutionGraph.execute(DefaultTaskExecutionGraph.java:186)\n' +
2021-09-14T21:20:35.5191390Z     '\tat org.gradle.execution.SelectedTaskExecutionAction.execute(SelectedTaskExecutionAction.java:39)\n' +
2021-09-14T21:20:35.5194562Z     '\tat org.gradle.execution.DefaultBuildWorkExecutor.execute(DefaultBuildWorkExecutor.java:40)\n' +
2021-09-14T21:20:35.5197473Z     '\tat org.gradle.execution.DefaultBuildWorkExecutor.access$000(DefaultBuildWorkExecutor.java:24)\n' +
2021-09-14T21:20:35.5200154Z     '\tat org.gradle.execution.DefaultBuildWorkExecutor$1.proceed(DefaultBuildWorkExecutor.java:48)\n' +
2021-09-14T21:20:35.5202992Z     '\tat org.gradle.execution.DryRunBuildExecutionAction.execute(DryRunBuildExecutionAction.java:49)\n' +
2021-09-14T21:20:35.5205956Z     '\tat org.gradle.execution.DefaultBuildWorkExecutor.execute(DefaultBuildWorkExecutor.java:40)\n' +
2021-09-14T21:20:35.5208880Z     '\tat org.gradle.execution.DefaultBuildWorkExecutor.execute(DefaultBuildWorkExecutor.java:33)\n' +
2021-09-14T21:20:35.5212653Z     '\tat org.gradle.execution.IncludedBuildLifecycleBuildWorkExecutor.execute(IncludedBuildLifecycleBuildWorkExecutor.java:36)\n' +
2021-09-14T21:20:35.5216410Z     '\tat org.gradle.execution.UndefinedBuildWorkExecutor.execute(UndefinedBuildWorkExecutor.java:48)\n' +
2021-09-14T21:20:35.5220106Z     '\tat org.gradle.execution.BuildOperationFiringBuildWorkerExecutor$ExecuteTasks.run(BuildOperationFiringBuildWorkerExecutor.java:57)\n' +
2021-09-14T21:20:35.5223878Z     '\tat org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:29)\n' +
2021-09-14T21:20:35.5227217Z     '\tat org.gradle.internal.operations.DefaultBuildOperationRunner$1.execute(DefaultBuildOperationRunner.java:26)\n' +
2021-09-14T21:20:35.5230546Z     '\tat org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:75)\n' +
2021-09-14T21:20:35.5233976Z     '\tat org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:68)\n' +
2021-09-14T21:20:35.5237602Z     '\tat org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:153)\n' +
2021-09-14T21:20:35.5241376Z     '\tat org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:68)\n' +
2021-09-14T21:20:35.5245005Z     '\tat org.gradle.internal.operations.DefaultBuildOperationRunner.run(DefaultBuildOperationRunner.java:56)\n' +
2021-09-14T21:20:35.5248729Z     '\tat org.gradle.internal.operations.DefaultBuildOperationExecutor.lambda$run$1(DefaultBuildOperationExecutor.java:71)\n' +
2021-09-14T21:20:35.5253414Z     '\tat org.gradle.internal.operations.UnmanagedBuildOperationWrapper.runWithUnmanagedSupport(UnmanagedBuildOperationWrapper.java:45)\n' +
2021-09-14T21:20:35.5257918Z     '\tat org.gradle.internal.operations.DefaultBuildOperationExecutor.run(DefaultBuildOperationExecutor.java:71)\n' +
2021-09-14T21:20:35.5262043Z     '\tat org.gradle.execution.BuildOperationFiringBuildWorkerExecutor.execute(BuildOperationFiringBuildWorkerExecutor.java:42)\n' +
2021-09-14T21:20:35.5265700Z     '\tat org.gradle.initialization.DefaultGradleLauncher.runWork(DefaultGradleLauncher.java:260)\n' +
2021-09-14T21:20:35.5269001Z     '\tat org.gradle.initialization.DefaultGradleLauncher.doClassicBuildStages(DefaultGradleLauncher.java:173)\n' +
2021-09-14T21:20:35.5272533Z     '\tat org.gradle.initialization.DefaultGradleLauncher.doBuildStages(DefaultGradleLauncher.java:149)\n' +
2021-09-14T21:20:35.5275905Z     '\tat org.gradle.initialization.DefaultGradleLauncher.executeTasks(DefaultGradleLauncher.java:125)\n' +
2021-09-14T21:20:35.5278883Z     '\tat org.gradle.internal.invocation.GradleBuildController$1.create(GradleBuildController.java:71)\n' +
2021-09-14T21:20:35.5281914Z     '\tat org.gradle.internal.work.DefaultWorkerLeaseService.withLocks(DefaultWorkerLeaseService.java:213)\n' +
2021-09-14T21:20:35.5285301Z     '\tat org.gradle.internal.invocation.GradleBuildController.doBuild(GradleBuildController.java:67)\n' +
2021-09-14T21:20:35.5288366Z     '\tat org.gradle.internal.invocation.GradleBuildController.run(GradleBuildController.java:56)\n' +
2021-09-14T21:20:35.5291588Z     '\tat org.gradle.tooling.internal.provider.ExecuteBuildActionRunner.run(ExecuteBuildActionRunner.java:31)\n' +
2021-09-14T21:20:35.5294822Z     '\tat org.gradle.launcher.exec.ChainingBuildActionRunner.run(ChainingBuildActionRunner.java:35)\n' +
2021-09-14T21:20:35.5298418Z     '\tat org.gradle.launcher.exec.BuildOutcomeReportingBuildActionRunner.run(BuildOutcomeReportingBuildActionRunner.java:66)\n' +
2021-09-14T21:20:35.5302562Z     '\tat org.gradle.tooling.internal.provider.ValidatingBuildActionRunner.run(ValidatingBuildActionRunner.java:32)\n' +
2021-09-14T21:20:35.5306928Z     '\tat org.gradle.tooling.internal.provider.FileSystemWatchingBuildActionRunner.run(FileSystemWatchingBuildActionRunner.java:90)\n' +
2021-09-14T21:20:35.5311755Z     '\tat org.gradle.launcher.exec.BuildCompletionNotifyingBuildActionRunner.run(BuildCompletionNotifyingBuildActionRunner.java:41)\n' +
2021-09-14T21:20:35.5316231Z     '\tat org.gradle.launcher.exec.RunAsBuildOperationBuildActionRunner$3.call(RunAsBuildOperationBuildActionRunner.java:49)\n' +
2021-09-14T21:20:35.5320170Z     '\tat org.gradle.launcher.exec.RunAsBuildOperationBuildActionRunner$3.call(RunAsBuildOperationBuildActionRunner.java:44)\n' +
2021-09-14T21:20:35.5324208Z     '\tat org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:200)\n' +
2021-09-14T21:20:35.5328361Z     '\tat org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:195)\n' +
2021-09-14T21:20:35.5332113Z     '\tat org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:75)\n' +
2021-09-14T21:20:35.5335460Z     '\tat org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:68)\n' +
2021-09-14T21:20:35.5339074Z     '\tat org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:153)\n' +
2021-09-14T21:20:35.5342840Z     '\tat org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:68)\n' +
2021-09-14T21:20:35.5346556Z     '\tat org.gradle.internal.operations.DefaultBuildOperationRunner.call(DefaultBuildOperationRunner.java:62)\n' +
2021-09-14T21:20:35.5350287Z     '\tat org.gradle.internal.operations.DefaultBuildOperationExecutor.lambda$call$2(DefaultBuildOperationExecutor.java:76)\n' +
2021-09-14T21:20:35.5355180Z     '\tat org.gradle.internal.operations.UnmanagedBuildOperationWrapper.callWithUnmanagedSupport(UnmanagedBuildOperationWrapper.java:54)\n' +
2021-09-14T21:20:35.5359861Z     '\tat org.gradle.internal.operations.DefaultBuildOperationExecutor.call(DefaultBuildOperationExecutor.java:76)\n' +
2021-09-14T21:20:35.5363866Z     '\tat org.gradle.launcher.exec.RunAsBuildOperationBuildActionRunner.run(RunAsBuildOperationBuildActionRunner.java:44)\n' +
2021-09-14T21:20:35.5367747Z     '\tat org.gradle.launcher.exec.InProcessBuildActionExecuter.lambda$execute$0(InProcessBuildActionExecuter.java:59)\n' +
2021-09-14T21:20:35.5370915Z     '\tat org.gradle.composite.internal.DefaultRootBuildState.run(DefaultRootBuildState.java:86)\n' +
2021-09-14T21:20:35.5374106Z     '\tat org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProcessBuildActionExecuter.java:58)\n' +
2021-09-14T21:20:35.5377595Z     '\tat org.gradle.launcher.exec.InProcessBuildActionExecuter.execute(InProcessBuildActionExecuter.java:30)\n' +
2021-09-14T21:20:35.5382025Z     '\tat org.gradle.launcher.exec.BuildTreeScopeLifecycleBuildActionExecuter.lambda$execute$0(BuildTreeScopeLifecycleBuildActionExecuter.java:34)\n' +
2021-09-14T21:20:35.5385753Z     '\tat org.gradle.internal.buildtree.BuildTreeState.run(BuildTreeState.java:53)\n' +
2021-09-14T21:20:35.5389648Z     '\tat org.gradle.launcher.exec.BuildTreeScopeLifecycleBuildActionExecuter.execute(BuildTreeScopeLifecycleBuildActionExecuter.java:33)\n' +
2021-09-14T21:20:35.5395079Z     '\tat org.gradle.launcher.exec.BuildTreeScopeLifecycleBuildActionExecuter.execute(BuildTreeScopeLifecycleBuildActionExecuter.java:28)\n' +
2021-09-14T21:20:35.5399857Z     '\tat org.gradle.tooling.internal.provider.ContinuousBuildActionExecuter.execute(ContinuousBuildActionExecuter.java:104)\n' +
2021-09-14T21:20:35.5404221Z     '\tat org.gradle.tooling.internal.provider.ContinuousBuildActionExecuter.execute(ContinuousBuildActionExecuter.java:55)\n' +
2021-09-14T21:20:35.5408763Z     '\tat org.gradle.tooling.internal.provider.SubscribableBuildActionExecuter.execute(SubscribableBuildActionExecuter.java:64)\n' +
2021-09-14T21:20:35.5413451Z     '\tat org.gradle.tooling.internal.provider.SubscribableBuildActionExecuter.execute(SubscribableBuildActionExecuter.java:37)\n' +
2021-09-14T21:20:35.5418701Z     '\tat org.gradle.tooling.internal.provider.SessionScopeLifecycleBuildActionExecuter.lambda$execute$0(SessionScopeLifecycleBuildActionExecuter.java:54)\n' +
2021-09-14T21:20:35.5422853Z     '\tat org.gradle.internal.session.BuildSessionState.run(BuildSessionState.java:67)\n' +
2021-09-14T21:20:35.5427038Z     '\tat org.gradle.tooling.internal.provider.SessionScopeLifecycleBuildActionExecuter.execute(SessionScopeLifecycleBuildActionExecuter.java:50)\n' +
2021-09-14T21:20:35.5432831Z     '\tat org.gradle.tooling.internal.provider.SessionScopeLifecycleBuildActionExecuter.execute(SessionScopeLifecycleBuildActionExecuter.java:36)\n' +
2021-09-14T21:20:35.5438179Z     '\tat org.gradle.tooling.internal.provider.GradleThreadBuildActionExecuter.execute(GradleThreadBuildActionExecuter.java:36)\n' +
2021-09-14T21:20:35.5442880Z     '\tat org.gradle.tooling.internal.provider.GradleThreadBuildActionExecuter.execute(GradleThreadBuildActionExecuter.java:25)\n' +
2021-09-14T21:20:35.5447840Z     '\tat org.gradle.tooling.internal.provider.StartParamsValidatingActionExecuter.execute(StartParamsValidatingActionExecuter.java:59)\n' +
2021-09-14T21:20:35.5453040Z     '\tat org.gradle.tooling.internal.provider.StartParamsValidatingActionExecuter.execute(StartParamsValidatingActionExecuter.java:31)\n' +
2021-09-14T21:20:35.5458302Z     '\tat org.gradle.tooling.internal.provider.SessionFailureReportingActionExecuter.execute(SessionFailureReportingActionExecuter.java:58)\n' +
2021-09-14T21:20:35.5463763Z     '\tat org.gradle.tooling.internal.provider.SessionFailureReportingActionExecuter.execute(SessionFailureReportingActionExecuter.java:42)\n' +
2021-09-14T21:20:35.5468568Z     '\tat org.gradle.tooling.internal.provider.SetupLoggingActionExecuter.execute(SetupLoggingActionExecuter.java:47)\n' +
2021-09-14T21:20:35.5472686Z     '\tat org.gradle.tooling.internal.provider.SetupLoggingActionExecuter.execute(SetupLoggingActionExecuter.java:31)\n' +
2021-09-14T21:20:35.5476268Z     '\tat org.gradle.launcher.daemon.server.exec.ExecuteBuild.doBuild(ExecuteBuild.java:65)\n' +
2021-09-14T21:20:35.5479288Z     '\tat org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:37)\n' +
2021-09-14T21:20:35.5482755Z     '\tat org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)\n' +
2021-09-14T21:20:35.5486446Z     '\tat org.gradle.launcher.daemon.server.exec.WatchForDisconnection.execute(WatchForDisconnection.java:39)\n' +
2021-09-14T21:20:35.5490066Z     '\tat org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)\n' +
2021-09-14T21:20:35.5493721Z     '\tat org.gradle.launcher.daemon.server.exec.ResetDeprecationLogger.execute(ResetDeprecationLogger.java:29)\n' +
2021-09-14T21:20:35.5497364Z     '\tat org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)\n' +
2021-09-14T21:20:35.5501452Z     '\tat org.gradle.launcher.daemon.server.exec.RequestStopIfSingleUsedDaemon.execute(RequestStopIfSingleUsedDaemon.java:35)\n' +
2021-09-14T21:20:35.5505617Z     '\tat org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)\n' +
2021-09-14T21:20:35.5508988Z     '\tat org.gradle.launcher.daemon.server.exec.ForwardClientInput$2.create(ForwardClientInput.java:78)\n' +
2021-09-14T21:20:35.5511951Z     '\tat org.gradle.launcher.daemon.server.exec.ForwardClientInput$2.create(ForwardClientInput.java:75)\n' +
2021-09-14T21:20:35.5514171Z     '\tat org.gradle.util.Swapper.swap(Swapper.java:38)\n' +
2021-09-14T21:20:35.5516537Z     '\tat org.gradle.launcher.daemon.server.exec.ForwardClientInput.execute(ForwardClientInput.java:75)\n' +
2021-09-14T21:20:35.5520083Z     '\tat org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)\n' +
2021-09-14T21:20:35.5523511Z     '\tat org.gradle.launcher.daemon.server.exec.LogAndCheckHealth.execute(LogAndCheckHealth.java:55)\n' +
2021-09-14T21:20:35.5527025Z     '\tat org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)\n' +
2021-09-14T21:20:35.5530239Z     '\tat org.gradle.launcher.daemon.server.exec.LogToClient.doBuild(LogToClient.java:63)\n' +
2021-09-14T21:20:35.5533169Z     '\tat org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:37)\n' +
2021-09-14T21:20:35.5536571Z     '\tat org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)\n' +
2021-09-14T21:20:35.5540431Z     '\tat org.gradle.launcher.daemon.server.exec.EstablishBuildEnvironment.doBuild(EstablishBuildEnvironment.java:84)\n' +
2021-09-14T21:20:35.5543997Z     '\tat org.gradle.launcher.daemon.server.exec.BuildCommandOnly.execute(BuildCommandOnly.java:37)\n' +
2021-09-14T21:20:35.5547397Z     '\tat org.gradle.launcher.daemon.server.api.DaemonCommandExecution.proceed(DaemonCommandExecution.java:104)\n' +
2021-09-14T21:20:35.5551177Z     '\tat org.gradle.launcher.daemon.server.exec.StartBuildOrRespondWithBusy$1.run(StartBuildOrRespondWithBusy.java:52)\n' +
2021-09-14T21:20:35.5554658Z     '\tat org.gradle.launcher.daemon.server.DaemonStateCoordinator$1.run(DaemonStateCoordinator.java:297)\n' +
2021-09-14T21:20:35.5557514Z     '\tat org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)\n' +
2021-09-14T21:20:35.5560214Z     '\tat org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:48)\n' +
2021-09-14T21:20:35.5562945Z     '\tat org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:56)\n' +
2021-09-14T21:20:35.5566954Z     'Caused by: org.gradle.workers.internal.DefaultWorkerExecutor$WorkExecutionException: A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintTask$AndroidLintLauncherWorkAction\n' +
2021-09-14T21:20:35.5570854Z     '\tat org.gradle.workers.internal.DefaultWorkerExecutor.lambda$submitWork$2(DefaultWorkerExecutor.java:208)\n' +
2021-09-14T21:20:35.5574352Z     '\tat org.gradle.internal.work.DefaultConditionalExecutionQueue$ExecutionRunner.runExecution(DefaultConditionalExecutionQueue.java:214)\n' +
2021-09-14T21:20:35.5578254Z     '\tat org.gradle.internal.work.DefaultConditionalExecutionQueue$ExecutionRunner.runBatch(DefaultConditionalExecutionQueue.java:164)\n' +
2021-09-14T21:20:35.5582042Z     '\tat org.gradle.internal.work.DefaultConditionalExecutionQueue$ExecutionRunner.run(DefaultConditionalExecutionQueue.java:131)\n' +
2021-09-14T21:20:35.5585274Z     '\tat org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:64)\n' +
2021-09-14T21:20:35.5587932Z     '\tat org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:48)\n' +
2021-09-14T21:20:35.5590616Z     '\tat org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:56)\n' +
2021-09-14T21:20:35.5593150Z     'Caused by: org.gradle.workers.WorkerExecutionException: There was a failure while executing work items\n' +
2021-09-14T21:20:35.5596354Z     '\tat org.gradle.workers.internal.DefaultWorkerExecutor.workerExecutionException(DefaultWorkerExecutor.java:270)\n' +
2021-09-14T21:20:35.5599951Z     '\tat org.gradle.workers.internal.DefaultWorkerExecutor.await(DefaultWorkerExecutor.java:252)\n' +
2021-09-14T21:20:35.5602896Z     '\tat org.gradle.workers.internal.NoIsolationWorkerFactory$1.lambda$execute$0(NoIsolationWorkerFactory.java:72)\n' +
2021-09-14T21:20:35.5605365Z     '\tat org.gradle.workers.internal.AbstractWorker$1.call(AbstractWorker.java:44)\n' +
2021-09-14T21:20:35.5607399Z     '\tat org.gradle.workers.internal.AbstractWorker$1.call(AbstractWorker.java:41)\n' +
2021-09-14T21:20:35.5610527Z     '\tat org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:200)\n' +
2021-09-14T21:20:35.5614656Z     '\tat org.gradle.internal.operations.DefaultBuildOperationRunner$CallableBuildOperationWorker.execute(DefaultBuildOperationRunner.java:195)\n' +
2021-09-14T21:20:35.5618437Z     '\tat org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:75)\n' +
2021-09-14T21:20:35.5621807Z     '\tat org.gradle.internal.operations.DefaultBuildOperationRunner$3.execute(DefaultBuildOperationRunner.java:68)\n' +
2021-09-14T21:20:35.5625357Z     '\tat org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:153)\n' +
2021-09-14T21:20:35.5629149Z     '\tat org.gradle.internal.operations.DefaultBuildOperationRunner.execute(DefaultBuildOperationRunner.java:68)\n' +
2021-09-14T21:20:35.5632807Z     '\tat org.gradle.internal.operations.DefaultBuildOperationRunner.call(DefaultBuildOperationRunner.java:62)\n' +
2021-09-14T21:20:35.5636658Z     '\tat org.gradle.internal.operations.DefaultBuildOperationExecutor.lambda$call$2(DefaultBuildOperationExecutor.java:76)\n' +
2021-09-14T21:20:35.5641456Z     '\tat org.gradle.internal.operations.UnmanagedBuildOperationWrapper.callWithUnmanagedSupport(UnmanagedBuildOperationWrapper.java:54)\n' +
2021-09-14T21:20:35.5646156Z     '\tat org.gradle.internal.operations.DefaultBuildOperationExecutor.call(DefaultBuildOperationExecutor.java:76)\n' +
2021-09-14T21:20:35.5650015Z     '\tat org.gradle.workers.internal.AbstractWorker.executeWrappedInBuildOperation(AbstractWorker.java:41)\n' +
2021-09-14T21:20:35.5653400Z     '\tat org.gradle.workers.internal.NoIsolationWorkerFactory$1.execute(NoIsolationWorkerFactory.java:59)\n' +
2021-09-14T21:20:35.5656387Z     '\tat org.gradle.workers.internal.DefaultWorkerExecutor.lambda$submitWork$2(DefaultWorkerExecutor.java:206)\n' +
2021-09-14T21:20:35.5658134Z     '\t... 6 more\n' +
2021-09-14T21:20:35.5660921Z     'Caused by: org.gradle.workers.internal.DefaultWorkerExecutor$WorkExecutionException: A failure occurred while executing com.android.build.gradle.internal.lint.AndroidLintWorkAction\n' +
2021-09-14T21:20:35.5664939Z     '\tat org.gradle.workers.internal.DefaultWorkerExecutor$WorkItemExecution.waitForCompletion(DefaultWorkerExecutor.java:342)\n' +
2021-09-14T21:20:35.5668915Z     '\tat org.gradle.internal.work.DefaultAsyncWorkTracker.waitForItemsAndGatherFailures(DefaultAsyncWorkTracker.java:142)\n' +
2021-09-14T21:20:35.5673701Z     '\tat org.gradle.internal.work.DefaultAsyncWorkTracker.waitForItemsAndGatherFailures(DefaultAsyncWorkTracker.java:90)\n' +
2021-09-14T21:20:35.5677617Z     '\tat org.gradle.internal.work.DefaultAsyncWorkTracker.waitForAll(DefaultAsyncWorkTracker.java:80)\n' +
2021-09-14T21:20:35.5681030Z     '\tat org.gradle.internal.work.DefaultAsyncWorkTracker.waitForCompletion(DefaultAsyncWorkTracker.java:68)\n' +
2021-09-14T21:20:35.5684198Z     '\tat org.gradle.workers.internal.DefaultWorkerExecutor.await(DefaultWorkerExecutor.java:250)\n' +
2021-09-14T21:20:35.5685897Z     '\t... 22 more\n' +
2021-09-14T21:20:35.5687013Z     'Caused by: java.lang.RuntimeException: Lint found errors in the project; aborting build.\n' +
2021-09-14T21:20:35.5689669Z     '\tat com.android.build.gradle.internal.lint.AndroidLintWorkAction.execute(AndroidLintWorkAction.kt:107)\n' +
2021-09-14T21:20:35.5692911Z     '\tat org.gradle.workers.internal.DefaultWorkerServer.execute(DefaultWorkerServer.java:63)\n' +
2021-09-14T21:20:35.5695779Z     '\tat org.gradle.workers.internal.NoIsolationWorkerFactory$1$1.create(NoIsolationWorkerFactory.java:66)\n' +
2021-09-14T21:20:35.5698640Z     '\tat org.gradle.workers.internal.NoIsolationWorkerFactory$1$1.create(NoIsolationWorkerFactory.java:62)\n' +
2021-09-14T21:20:35.5701871Z     '\tat org.gradle.internal.classloader.ClassLoaderUtils.executeInClassloader(ClassLoaderUtils.java:97)\n' +
2021-09-14T21:20:35.5705167Z     '\tat org.gradle.workers.internal.NoIsolationWorkerFactory$1.lambda$execute$0(NoIsolationWorkerFactory.java:62)\n' +
2021-09-14T21:20:35.5706893Z     '\t... 21 more\n' +
2021-09-14T21:20:35.5707448Z     '\n' +
2021-09-14T21:20:35.5707967Z     '\n' +
2021-09-14T21:20:35.5708848Z     '* Get more help at https://help.gradle.org\n' +
2021-09-14T21:20:35.5709645Z     '\n' +
2021-09-14T21:20:35.5710266Z     'BUILD FAILED in 5m 58s',
2021-09-14T21:20:35.5710775Z   failed: true,
2021-09-14T21:20:35.5711273Z   timedOut: false,
2021-09-14T21:20:35.5711804Z   isCanceled: false,
2021-09-14T21:20:35.5712318Z   killed: false,
2021-09-14T21:20:35.5713234Z   pluginName: '@semantic-release/exec'
2021-09-14T21:20:35.5713829Z }
2021-09-14T21:20:35.5812891Z Post job cleanup.
```

--- a/demo/build.gradle
+++ b/demo/build.gradle
@@ -64,7 +64,7 @@ dependencies {
 
     // [START_EXCLUDE silent]
     // implementation project(':library')
-    implementation 'androidx.appcompat:appcompat:1.3.1'
+    implementation 'androidx.appcompat:appcompat:1.4.0-alpha03'
     implementation 'androidx.lifecycle:lifecycle-extensions:2.2.0'
 
     // GMS

